comtypes/_meta.py

```diff
@@ -38,24 +41,29 @@ class _coclass_meta(type):
     # POINTER(...) type gets a __ctypes_from_outparam__ method which
     # will QueryInterface for the default interface: the first one on
     # the coclass' _com_interfaces_ list.
-    def __new__(cls, name, bases, namespace):
-        klass = type.__new__(cls, name, bases, namespace)
+    def __new__(mcs, name, bases, namespace):
```

Comment:
same as line 333 in `comtypes/__init__.py`

Reply:
look at where the method is declared. It is inside a metaclass so the use of mcs as the first parameter is correct.